packages/dialtone-vue2/components/tooltip/tooltip.test.js

```diff
@@ -107,6 +107,15 @@ describe('DtTooltip tests', () => {
         expect(tooltip.textContent).toBe('Test message');
       });
 
+      describe('When tooltip content is a space character', () => {
+        it('should not render the content', async () => {
+          wrapper.destroy();
```

Comment:
why are we destroying the wrapper here?

Reply:
I think it's only an issue in this case because of the tooltip writing to the body of the JSDom instance and that will persist between tests unless you destroy. It causes a bunch of errors if I don't do it. 

There is wrapper.destroy in afterEach but that doesn't get run when there are nested describes like this.

it runs the first beforeEach:
```
mockProps = { show: true };
updateWrapper();
```

then after in the test we do:
```
wrapper.destroy();
mockProps = { show: true, message: ' ' };
updateWrapper();
```

Seems to cause problems without the wrapper.destroy() because the previously mounted updateWrapper in the beforeEach hasn't been destroyed, and the jsdom instance is still there.

Reply:
might also be because we are getting the content like so:

```
.tippy-content
```

so if there are multiple tippys rendered it will just take the first one.